### PR TITLE
Add Uncrustify to the list

### PR DESCRIPTION
**Pull request by Swizz**
**Information:**
- Language: C, C++, Java, Objective C
- Linter: [uncrustify](https://github.com/uncrustify/uncrustify)
- URL: http://uncrustify.sourceforge.net

**Checklist:**
- [x] Only added one linter?
- [x] Is it inside the right language container?
- [x] Is it sorted alphabetically inside the container?
- [ ] Does the title follow the template:  "Add [LINTER] for [LANGUAGE]."? _3 languages_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ This project is not closed to actual static analyzers. With this repository we i
 ### C
 
 - [oclint](https://github.com/oclint/oclint) - Static analysis for C, C++ and Objective-C. Written in C++.
+- [uncrustify](https://github.com/uncrustify/uncrustify) - A source code beautifier for C, C++, C#, ObjectiveC, D, Java, Pawn and VALA
```
Comment (mlcdf) [inline README.md]: I'm nitpicking here, but would you mind removing the 'A' at the beginning and adding a point at the end of the sentence? :)

**Comment by Swizz** [comment]
Here we are :smiley:

**Comment by caramelomartins** [comment]
@mlcdf Looks good to me.

**Comment by caramelomartins** [comment]
@Swizz thanks for following @mlcdf 's advice!

**Comment by mlcdf** [comment]
Thank you for your contribution @Swizz !

**Pull request merged**